pkg/apis/serving/v1beta1/inference_service_status.go

```diff
@@ -116,6 +116,8 @@ const (
 	ExplainerReady apis.ConditionType = "ExplainerReady"
 	// IngressReady is set when Ingress is created
 	IngressReady apis.ConditionType = "IngressReady"
+	// ServiceReady is set when network configurations for all components have completed.
+	ServiceReady apis.ConditionType = "ServiceReady"
```

Comment:
Should we rename this to `RoutesReady` ? now I think the aggregation of `ConfigurationReady` conditions is actually the deployment status, the `Ready` condition is really the further aggregation of the two.